integration-tests/olminstall/helpers/kubearchive.py

```
@@ -0,0 +1,48 @@
+class KubeArchiveClient:
+    def __init__(self, host: str, token: str) -> None:
+        self.host = host.rstrip("/")
+        self.token = token
+        self.available: bool | None = None
+
+    def _request(self, path: str) -> str:
+        req = Request(
+            f"{self.host}{path}",
+            headers={"Authorization": f"Bearer {self.token}"},
+            method="GET",
+        )
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)